lms/djangoapps/course_api/blocks/transformers/extra_fields.py

```diff
@@ -0,0 +1,69 @@
+"""
+Extra Fields Transformer
+"""
+from django.conf import settings
+
+from openedx.core.djangoapps.content.block_structure.transformer import BlockStructureTransformer
+
+
+class ExtraFieldsTransformer(BlockStructureTransformer):
+    """
+    Shows ExtraFields specified in COURSE_BLOCKS_API_EXTRA_FIELDS LMS Django Settings variable
```

Comment:
Let's say instead: "A configurable transformer that adds additional XBlock fields to the course blocks API.

Extra fields must be specified using the COURSE_BLOCKS_API_EXTRA_FIELDS LMS Django settings variable. Open edX instances can use this to make additional XBlock fields available via the course blocks API, that aren't otherwise included by default."